tests/Functional/Driver/PDO/SQLSrv/DriverTest.php

```diff
@@ -77,4 +78,31 @@ public function testDriverOptions(): void
                 ->getAttribute(PDO::ATTR_CASE)
         );
     }
+
+    /**
+     * @dataProvider stringDataProvider
+     */
+    public function testDriverStringBinding(string $bindingType, string $expectedSqlType): void
+    {
+        $statement = $this->connection->prepare(
+            'SELECT sql_variant_property(:parameter, \'BaseType\') AS type'
```

Comment:
The usage of named parameters is unnecessary here. They are emulated for most of the drivers and extend the scope of the test.